sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/DecryptResult.java

```diff
@@ -0,0 +1,23 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+/**
+ * Represents the details of decrypt operation result.
+ */
+public class DecryptResult {
+    public DecryptResult(byte[] plainText) {
+        this.plainText = plainText;
+    }
+
+    /**
+     * The decrypted content.
+     */
+    private byte[] plainText;
```

Comment:
Conventionally, properties of a class are declared before methods.

Reply:
updated